files/metrics_tidy

```diff
@@ -55,3 +55,6 @@ find "$metrics_directory" -type f -ctime +"$retention_days" -delete
 # The return code of a pipeline is the rightmost command, which means we trigger our trap if tar fails
 find "$metrics_directory" -type f -name "*json" | \
   tar --create --gzip --file "${metrics_directory}/${metrics_type}-$(date +%Y.%m.%d.%H.%M.%S).tar.gz" --files-from -
+
+# Cleanup the backed up json files so that we do not duplicate files in the tarballs.
+find "$metrics_directory" -type f -name "*json" -delete
```

Comment:
I'm still not convinced running a separate `find` is the best way to do this since it can result in files being deleted before they are archived depending on how the cron jobs line up.

The old way of running find, saving the results to a file and then having `tar` and `rm` both operate on that file seemed to work well. Is there a reason we can't return to using that?

Reply:
I agree, my concerns about `xargs` are probably unfounded in this case.  Pushed a commit to this branch.